src/main/java/com/brightpath/learnify/domain/quiz/comment/Comment.java

```diff
@@ -0,0 +1,14 @@
+package com.brightpath.learnify.domain.quiz.comment;
+
+import com.brightpath.learnify.domain.common.ResourceType;
+
+import java.util.UUID;
+
+public record Comment(UUID id,
+                      String commentOwnerId,
+                      ResourceType resourceType,
```

Comment:
Minor: 
Do we need resourceType and resourceId at domain level? (I think probably only at the persistance level and in creation DTO)
But it could be usable if we would like to display all comments for specific user - we can leave that as it is